packaging/current_ebpf.version

```diff
@@ -0,0 +1,2 @@
+v0.7.9.1
+
```

Comment:
It looks like this is a spurious blank line.

Reply:
Thank you @Ferroin ! In reality this was mistake, I have the habit to change this file, but for this PR I have to change `libbpf`. The last commit fixed the problem.